doc/API.md

```diff
@@ -107,25 +100,36 @@ either `status` or deposit `outpoints`.
 | ------------- | ------------------------------------------ | ------------------------- |
 | `vaults`      | array of [vault resource](#vault-resource) | Vaults filtered by status |
 
-### `getonchaintxs`
 
-The `getonchaintxs` RPC Command retrieves the on chain transactions of a
-vault with the given deposit `outpoint`.
+### `listtransactions`
 
-#### Request
+| Parameter   | Type         | Description                                                                                     |
+| ----------- | ------------ | ----------------------------------------------------------------------------------------------- |
+| `outpoints` | string array | Vault IDs -- optional, filter the list with the given vault Outpoints                           |
```

Comment:
in this PR, the listtransactions command as an outpoint in its params which is required

Reply:
Right, the doc is wrong thanks :+1:

Reply:
I think the doc is right :)

Reply:
Lol, it was at the time of writing but now that we agreed it effectively is!